src/security/tpm/Makefile.inc

```diff
@@ -55,10 +55,22 @@ romstage-y += tspi/crtm.c
 ramstage-y += tspi/crtm.c
 postcar-y += tspi/crtm.c
 
+ifeq ($(TPM_MEASURED_BOOT_SPEC_LOG),y)
```

Comment:
Shouldn't this be reversed?

Reply:
Of course.. it also lacks `CONFIG_` prefix which is why I didn't notice wrong condition.